examples/IRMQTTServer/IRMQTTServer.ino

```diff
@@ -515,18 +517,19 @@ void saveWifiConfigCallback(void) {
   flagSaveWifiConfig = true;
 }
 
-// Forcibly mount the SPIFFS. Formatting the SPIFFS if needed.
+// Forcibly mount the FILESYSTEM. Formatting the FILESYSTEM if needed.
 //
 // Returns:
 //   A boolean indicating success or failure.
 bool mountSpiffs(void) {
-  debug("Mounting SPIFFS...");
-  if (SPIFFS.begin()) return true;  // We mounted it okay.
+  debug("Mounting FILESYSTEM...");
```

Comment:
`  debug("Mounting " FILESYSTEM "...");`
Use that instead, and it should report if it is SPIFFS or LittleFS etc.

Reply:
Already change to:
```c
#if (FILESYSTEM == LittleFS)
    debug("... LittleFS ...");
#else
    debug("... SPIFSS ..."0;
#endif
```

Reply:
If you use my suggestion, then you don't need the `#if` clause and it works for any other type of FS.